Source/Core/DolphinWX/Debugger/CodeWindowFunctions.cpp

```diff
@@ -202,6 +204,22 @@ void CCodeWindow::OnSymbolsMenu(wxCommandEvent& event)
     NotifyMapLoaded();
     break;
   }
+  case IDM_SCAN_RSO:
+  {
+    RSOChainView rso_chain;
+    wxTextEntryDialog dialog(this, _("Enter RSO module address:"), wxGetTextFromUserPromptStr,
+                             wxEmptyString);
+    if (dialog.ShowModal() == wxID_OK)
+    {
+      u64 address = strtoul(dialog.GetValue().ToStdString().c_str(), nullptr, 0);
```

Comment:
wxString has [ToULong](http://docs.wxwidgets.org/3.1.0/classwx_string.html#a36a50d60b6d2ff7a08f112c8db8fc109) and [ToCULong](http://docs.wxwidgets.org/3.1.0/classwx_string.html#a545b481942916202dacaaee5e558d035) (for enforcing classic locale) functions. The previously mentioned functions should likely also be used to sanitize against invalid inputs. As far as I can tell, if I entered a random set of text, like "zhfwksk", it would continue and load from the address 0 (as strtoul returns 0 in the event no valid conversion could be done).